### ISSUE TITLE
Traffic Ops new post-install failed (1.1.5)

### ISSUE THREAD

**Issue by smalenfant** [Comcast/traffic_control#521]
During the postinstall, the following occurred. Is the problem with the LogFormat.Format ?

```
inserting profile parameter value: profile_name = EDGE1_CDN_520 parameter_name = CONFIG proxy.config.exec_thread.limit config_file = records.config value = INT 32
inserting profile parameter value: profile_name = EDGE1_CDN_520 parameter_name = CONFIG proxy.config.allocator.thread_freelist_size config_file = records.config value = INT 1024
inserting profile parameter value: profile_name = EDGE1_CDN_520 parameter_name = CONFIG proxy.config.mlock_enabled config_file = records.config value = INT 2
inserting profile parameter value: profile_name = EDGE1_CDN_520 parameter_name = LogFormat.Format config_file = logs_xml.config value = %<cqtq> chi=%<chi> phn=%<phn> shn=%<shn> url=%<cquuc> cqhm=%<cqhm> cqhv=%<cqhv> pssc=%<pssc> ttms=%<ttms> b=%<pscl> sssc=%<sssc> sscl=%<sscl> cfsc=%<cfsc> pfsc=%<pfsc> crc=%<crc> phr=%<phr> uas="%<{User-Agent}cqh>"
The insert failed
panic: Error 1452: Cannot add or update a child row: a foreign key constraint fails (`traffic_ops_db`.`profile_parameter`, CONSTRAINT `fk_atsprofile_atsparameters_atsparameters1` FOREIGN KEY (`parameter`) REFERENCES `parameter` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION)

goroutine 1 [running]:
main.main()
	/vol1/jenkins/jobs/traffic_ops/workspace/traffic_ops/install/go/src/comcast.com/dataload/dataload.go:233 +0x83fc

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 5 [chan receive]:
database/sql.(*DB).connectionOpener(0xc82007eaa0)
	/usr/local/go/src/database/sql/sql.go:634 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:481 +0x336
failed to load seed data.
```

**Comment by smalenfant** [comment]
I removed the LogFormat.Format line and installation continues.

**Comment by limited** [comment]
What specific line(s) did you remove? I tried taking the two LogFormat.Format lines out of the traffic_control/traffic_ops/install/data/json/parameter.json, but didn't have any luck getting installation to move forward

**Comment by smalenfant** [comment]
I removed that one

`LogFormat.Format config_file = logs_xml.config value = %<cqtq> chi=%<chi> phn=%<phn> shn=%<shn> url=%<cquuc> cqhm=%<cqhm> cqhv=%<cqhv> pssc=%<pssc> ttms=%<ttms> b=%<pscl> sssc=%<sssc> sscl=%<sscl> cfsc=%<cfsc> pfsc=%<pfsc> crc=%<crc> phr=%<phr> uas="%<{User-Agent}cqh>"`

**Comment by limited** [comment]
I had to remove this line as well to get TO to install:
{"profile":"EDGE1_CDN_520","parameter":"cron_ort_syncds","config_file":"traffic_ops_ort_syncds.cron","value":"*/15 * * * * root /opt/ort/traffic_ops_ort.pl syncds warn https://tm.kabletown.net admin:password > /tmp/ort/syncds.log 2>&1"}

**Comment by dangogh** [comment]
This does not seem to be an issue any more.  Can you try again with 1.1.7?

**Comment by jianli123** [comment]
Any update on the 1.1.7 for the postinstall?

**Comment by smalenfant** [comment]
Trying to build a DEV box to test it. Unless a test RPM can be released.

**Comment by dewrich** [comment]
@smalenfant Working through trying to close issues, this still a problem?

**Comment by dangogh** [comment]
was fixed prior to 1.2.0,   but broken again due to cdn_name change.  #659 fixed that problem for 1.2.1.

**Comment by smalenfant** [comment]
I don't think that was ever resolved prior to 1.2.1. Had to do with parameter.json not matching with profile_parameter.json. Tested in 1.2.1 RC1
